test/e2e/node_feature_discovery_test.go

```diff
@@ -873,6 +886,64 @@ core:
 				})
 			})
 
+			// Test NodeFeatureGroupRule
+			Context("and NodeFeatureGroupRules objects deployed", func() {
+				BeforeEach(func(ctx context.Context) {
+					// We need a NodeFeature from the node, can't be a fake one
+					if !useNodeFeatureApi {
+						Skip("NodeFeature API not enabled")
+					}
+				})
+				It("custom NodeFeatureGroup from the NodeFeatureGroupRule rules should be created", func(ctx context.Context) {
```

Comment:
Doesn't look right, e.g.

```suggestion
				It("NodeFeatureGroup should be updated", func(ctx context.Context) {
```

??